sentences/hr/_common.yaml

```diff
@@ -1,7 +1,7 @@
 language: hr
 responses:
   errors:
-    no_intent: "Nažalost, nisam vas razumio"
+    no_intent: "Nažalost, ne razumijem (zahtjev|pitanje)"
```

Comment:
I don't think variations are available in errors, just choose one. Sorry for confusion.